config/application.rb

```diff
@@ -60,7 +60,8 @@ class Application < Rails::Application
     config.export_batch_size = ENV["EXPORT_SERVICE_BATCH_SIZE"] || 1000
 
     config.years_before_expiry = ENV["YEARS_BEFORE_EXPIRY"] || 3
-    config.ad_letters_exports_expires_in = ENV["AD_LETTERS_EXPORT_EXPIRES_IN"] || 35
+    config.ad_letters_exports_expires_in = ENV["AD_LETTERS_EXPORT_EXPIRES_IN"] || 35 # Days
+    config.ad_letters_delete_records_in = ENV["AD_LETTERS_DELETE_RECORDS_IN"] || 3 # Weeks
```

Comment:
Can we make this `days`? I understand that for production we have been specifying 3 weeks, but we can just as easily represent that as `21 # Days`.

My reasoning is to give us flexibility in non-production environments for testing.